tests/utils/snapPoints.js

```diff
@@ -39,7 +39,32 @@ const isAtPercentageSnapPoint = async (browser, cardSelector, expectedSnapPointP
     return true
 }
 
+const isAtSnapPoint = async (browser, cardSelector, snapPointValue)=>{
```

Comment:
💅🏽  What do you think about this naming?
* `verifyCardPosition` 
* `verifyCardRelativePosition` (percentage)
* `verifyCardAbsolutePosition` (number)